solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/DesignSystem/DS+Colors.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DS+Colors.swift
+//  SpaceApp
+//
+//  Created by Pedro Gabriel on 14/03/22.
+//
+
+import SwiftUI
+
+extension DS {
+    public struct Color {
+        let color: SwiftUI.Color
+    }
+
+    public struct DynamicColor {
+        @Environment(\.colorScheme) private var colorScheme
+
+        let light: SwiftUI.Color
+        let dark: SwiftUI.Color
+
+        var dsColor: DS.Color {
+            colorScheme == .light ? .init(color: light) : .init(color: dark)
+        }
+    }
+}
+
+extension DS.Color {
+    /// A white color suitable for use in UI elements.
+    @available(*, deprecated, message: "Check if you should use primary, secondary, background or accentColor")
+    public static let white: Self = .init(color: .white)
```

Comment:
**[CHANGE REQUEST]**
Avoid color names on DS colors definitions. In a design system we have a nome of a color related to its context or use. So instead of `white` use directly `primary`, `secondary` or some other named color.
The api consumer does not need to know if the color is white, black or whatever, it needs to know its context and/or use instead... So, if you call this white then the definition changes later on, you can have `white` but it's end result is actually something else.

Reply:
Removed static colors such as white and blue.